Swiftris/Swiftris/GameViewController.swift

```diff
@@ -104,6 +118,29 @@ class GameViewController: UIViewController, SwiftrisDelegate, UIGestureRecognize
         }
         swiftris.dropShape()
     }
+    
+    // Notification
+    
+    func swiftrisDidEnterBackground() {  // read with NSNotification
+        if !isPause {
+            isPause = true
+            scene.stopTicking()
+            player.pause()
+            timer.invalidate()
+        NSLog("From GameViewController: swiftrisDidEnterBackground")
+        }
+    }
+    
+    func swiftrisDidEnterForeground() {  // read with NSNotification
```

Comment:
You may want to reconsider un-pausing the game automatically when the application becomes active again. This may take the user by surprise and end up costing them the game. Instead, I think it would be better to allow the user to un-pause the game when they are ready.